git-ftp

```diff
@@ -1082,6 +1129,20 @@ handle_fetch() {
 # ------------------------------------------------------------
 # Actions
 # ------------------------------------------------------------
+
+action_clone() {
+    set_up_clone_folder 
+	
+    check_lftp_available
```

Comment:
Maybe do this first?

Otherwise this fails:
```sh
git ftp clone $url
# error, but directory created
apt-get install lftp
git ftp clone $url
# error, directory exists already
```